src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -141,31 +145,63 @@ export const TradeInput = memo(() => {
   const slippageDecimal = useAppSelector(selectTradeSlippagePercentageDecimal)
   const activeQuoteErrors = useAppSelector(selectActiveQuoteErrors)
   const quoteRequestErrors = useAppSelector(selectTradeQuoteRequestErrors)
+  const quoteResponseErrors = useAppSelector(selectTradeQuoteResponseErrors)
   const isUnsafeQuote = useAppSelector(selectIsUnsafeActiveQuote)
 
+  // whenever the sell amount changes, set the quote states to fetching
+  useEffect(() => {
+    setIsAnySwapperFetched(false)
+    setIsQuoteRequestComplete(false)
+  }, [sellAmountCryptoPrecision])
+
+  // when the a quote becomes available, set the quote state to fetched
```

Comment:
Since this refers to holistic swapper fetched status
```suggestion
  // when the a quote becomes available, set the quote states to fetched
```